README.md

```diff
@@ -11,30 +12,41 @@ These are served via two configurable domain names, such as `docker.example.com`
 Requirements
 ============
 
-Python3 on the host is strongly recommended, as Ansible with Python 2 is deprecated and could break at any point.
-
-Hardware requirements for the instance are fairly low, requiring simply 1 core and 2GB RAM at a minimum. Disk requirements will vary depending on the number of images cached. 
-
-In testing mirroring all components for a Magnum cluster deployment required 30-70GB depending on mirrored version granularity.
-
-Naturally these requirements will scale with the number of users: in testing a single user doing a docker pull over LAN will consume 1 CPU for TCP handling. Thus, for 10 concurrent pulls 12+ cores will be required (to allow for various host overheads).
+- Kubernetes cluster 1.10+ (can use Cluster API)
+- Helm 2.8.0+
+- High available ingress controller
```

Comment:
This uses a load balancer instead of a HA ingress controller. We should either remove this line or put a `TODO` into it

Reply:
Can I substitute it with "- Loadbalancers for Longhorn, Postgres, Kubernetes Cluster and Harbor Service" ?